src/test/java/org/openrewrite/analysis/controlflow/ControlFlowTest.java

```diff
@@ -2739,4 +2799,43 @@ void test(E x) /*~~(BB: 7 CN: 3 EX: 1 | 1L)~~>*/{
           )
         );
     }
+
+    @Test
+    void switchCaseNoBreakOrReturn() {
```

Comment:
Would you be willing to upload an image of the graph that gets generated for this test?

Also, could you add a test for a single case/switch statement with no additional cases as well?

Reply:
Ah, I think you're right (you can debate what's better or worse though, having a non-full graph versus a break recipe 😅). Anyway, the graph of the `switchCaseNoBreakOrReturn ` test looks like:

![image](https://github.com/user-attachments/assets/b4d38dc6-aeea-42dc-92a4-3eda9929b928)

As I am not very familiar with control flow visualizations, I am wondering how the graph should look in the first place. I guess to draw the full graph, there should be one more arrow going from the `System.out.println(“B”)` block to the `System.out.println(“Done”)` block, right?

Reply:
Yes, you're correct. That should be how the graph looks. Do you think you understand control flow well enough to fix it?

Those exceptions I put in place were to internally verify the graph was well-formed while I was building it. It would be good to restore them